dnsaas/api/v2/serializers.py

```diff
@@ -115,6 +126,27 @@ def _clean_txt_content(self, record_type, attrs):
         if record_type == 'TXT':
             attrs['content'] = attrs['content'].replace('\\', '')
 
+    def _validate_service(self, attrs):
+        if (
+            # enforce service required for patch updates
```

Comment:
actually:
post and patch:
https://github.com/allegro/django-powerdns-dnssec/blob/8adce140d3da0acbd395536201fb5c5a347fd02f/ui/static/app/record/record.service.ts#L31

;p